static/docs/command-reference/run.md

```diff
@@ -52,6 +52,14 @@ captures data and <abbr>caches</abbr> relevant <abbr>data artifacts</abbr> along
 the way. See [this example](/doc/get-started/example-pipeline) to learn more and
 try creating a pipeline.
 
+### Parallel execution
+
+When running your command, DVC will remove the project lock (`.dvc/lock` file),
+so that you will be able to run other DVC commands in parallel. However, it uses
+per-path read-write locking instead, to guarantee that no two DVC instances
```

Comment:
are those locks file-based? do we need to clarify how to find and clean them in case something happens? (it though can be part of the new supposed-to-be troubleshooting guide)